crates/ott-balancer/src/balancer.rs

```diff
@@ -202,7 +202,7 @@ impl Default for BalancerContext {
             monoliths: HashMap::default(),
             rooms_to_monoliths: HashMap::default(),
             monoliths_by_region: HashMap::default(),
-            monolith_selection: Box::<MinRoomsSelector>::default(),
+            monolith_selection: BalancerConfig::get().selection_strategy,
```

Comment:
This shouldn't be done in the `Default` impl. `Default` should return the exact same value every time, eg the "default" value. I would recommend deriving `Default` for `MonolithSelectionStrategy`. `BalancerConfig::get()` can panic if the config is not initialized (which is why the ci is failing)

Reply:
See: https://doc.rust-lang.org/std/default/trait.Default.html#enums

Reply:
Okay, makes sense